lib/services/project-service.ts

```diff
@@ -111,6 +112,7 @@ export class ProjectService implements IProjectService {
 
 			this.$projectDataService.initialize(projectDir);
 			this.$projectDataService.setValue("id", projectId).wait();
+			this.$npm.executeNpmCommand("npm install " + constants.TNS_CORE_MODULES_NAME + " --save --save-exact", projectDir).wait();
```

Comment:
I think yes, if we want to specify the exact number and prevent change on `npm update`. Perhaps it's a good idea to add an optional parameter on `create` so that the user can pass the version. In both cases they will be able to edit the package.json and change the version.